user/clight/template.py

```diff
@@ -0,0 +1,36 @@
+pkgname = "clight"
+pkgver = "4.11"
+pkgrel = 0
+build_style = "cmake"
+hostmakedepends = [
+    "cmake",
+    "dbus-devel",
+    "elogind-devel",
+    "gsl-devel",
+    "libconfig-devel",
+    "libmodule-devel",
+    "linux-headers",
+    "ninja",
+    "pkgconf",
+    "popt-devel",
+]
+makedepends = [
```

Comment:
small lists like this can and should be one line

Reply:
I did run `ruff format` (on whole repo) - whatever it did is here, on PR.

Reply:
Didn't understood which list do you mean - makedepends or hostmakedepends? Please make a comment at least two or three lines *below* the line you're commenting. In any case, that's just more readable as it follows generic template.